mmv1/third_party/terraform/utils/common_diff_suppress.go.erb

```diff
@@ -179,3 +179,8 @@ func durationDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	}
 	return oDuration == nDuration
 }
+
+// Suppress all diffs, used for Disk.Interface which is a nonfunctional field
+func alwaysDiffSuppress(_, _, _ string, _ *schema.ResourceData) bool {
+	return true
```

Comment:
The behaviour of a DSF like this surprised me, so I figure I'll note it- whatever value is stored in state will always be returned. That seems fine here, although we'll need to clear out user states in `4.0.0` when we remove the field using a stateupgrader.

Reply:
Yeah, it locks in whatever exists in state which allows the user to remove the field from their config without triggering recreate (as the entire resource is `input`/ForceNew)